packages/plugins/content/image/src/Component/Form/index.js

```diff
@@ -2,14 +2,23 @@
 import React from 'react'
 import Display from '../Display'
 import TextField from 'material-ui/TextField'
+import Checkbox from 'material-ui/Checkbox'
 import type { PropTypes } from '../index.js'
 
 import { BottomToolbar } from 'ory-editor-ui'
 
 const handleChange = (onChange: Function) => (e: Event) => {
   const target = e.target
   if (target instanceof HTMLInputElement) {
-    onChange({ src: target.value })
+    const change = {}
+
+    if (target.type === 'checkbox') {
+      change[target.name] = target.checked ? '_blank' : '';
```

Comment:
The issue with `_blank` links is with security, please read more about this [here](https://mathiasbynens.github.io/rel-noopener/) and consider adding the rel attribute!